tools/archetype/binding.test/src/main/resources/archetype-resources/META-INF/MANIFEST.MF

```diff
@@ -1,31 +1,30 @@
-Manifest-Version: 1.0
-Bundle-ManifestVersion: 2
-Bundle-Name: ${bindingIdCamelCase} Binding Tests
-Bundle-RequiredExecutionEnvironment: JavaSE-1.8
-Bundle-SymbolicName: ${artifactId};singleton:=true
-Bundle-Vendor: ${vendorName}
-Bundle-Version: ${version.replaceAll("-SNAPSHOT", ".qualifier")}
-Export-Package: ${package};uses:="org.eclipse.smarthome.test"
-Fragment-Host: ${package}
-Import-Package: 
- ${package},
- ${package}.handler,
- org.eclipse.jdt.annotation;resolution:=optional,
- org.eclipse.smarthome.core.common.registry,
- org.eclipse.smarthome.core.events,
- org.eclipse.smarthome.core.thing,
- org.eclipse.smarthome.core.thing.binding,
- org.eclipse.smarthome.core.thing.binding.builder,
- org.eclipse.smarthome.core.thing.util,
- org.eclipse.smarthome.test.java,
- org.eclipse.smarthome.test.storage,
- org.hamcrest;core=split,
- org.hamcrest.collections,
- org.hamcrest.core,
- org.junit;version="4.0.0",
- org.mockito,
- org.mockito.invocation,
- org.mockito.stubbing,
- org.osgi.framework,
- org.osgi.service.device,
- org.slf4j
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: ${bindingIdCamelCase} Binding Tests
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-SymbolicName: ${artifactId};singleton:=true
+Bundle-Vendor: ${vendorName}
+Bundle-Version: ${version.replaceAll("-SNAPSHOT", ".qualifier")}
+Export-Package: ${package};uses:="org.eclipse.smarthome.test"
+Fragment-Host: ${package}
+Import-Package:
```

Comment:
Can you readd the space at the end of this line?
Without the space the build fails with: `Error assembling JAR: invalid header field`